frontend/test/utils.test.ts

```diff
@@ -144,4 +148,22 @@ describe('Utilities', () => {
       expect(roundReceivableAmountAfterFees(finalAmount, fee, eth)).toEqual('292.05');
     });
   });
+  describe('humanizeMinSendAmount', () => {
+    const tests = [
+     {input: 123, output: 120},
+     {input: 1235, output: 1200},
+     {input: 1295, output: 1300},
+     {input: 432498, output: 430000},
+     {input: 4.8956, output: 5},
+     {input: 4.1956, output: 4},
+     {input: 0.1956, output: 0.2},
+     {input: 0.0194, output: 0.019},
```

Comment:
I'm curious if anyone disagrees with these expectations RE how we round minSend amounts for users

Reply:
This seems fine to me! @apbendi any thoughts?

Reply:
Actually right now on Optimism it says the minimum send is `0.0097 OETH`, but that should probably be rounded to `0.01 OETH`?

Reply:
Though maybe not, If it was 0.0051 OETH we wouldn't want to round that up. In general, I guess it just feels like the variable min-send UX is kinda confusing. Gas prices might be at a low when I send funds, then they're basically stuck. It seems better to have a consistent min send based on e.g. the 90th percentile gas price, so the floor is constant and 90% of the time we know users will be able to withdraw. And then we just periodically update that.

But, this can be discussed in a separate issue

Reply:
Ahh got it, sounds good. Yes I'll resolve now!